src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -217,3 +217,9 @@ import { ViewTransitions } from 'astro:transitions';
 :::note[known limitations]
 The `morph` animation cannot be simulated in traditional CSS. So any element using this animation will not be animated.
 :::
+
+## `prefers-reduced-motion`
+
+You may be concerned about how all these animations will affect users that have [`prefer-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) enabled on their device. Don't worry! Currently, Astro includes a CSS media query that disables *all* view transition animations whenever this setting is detected. Instead, the browser will simply swap the DOM elements without an animation.
```

Comment:
This reads to me a little more like the writing style in a blog article (a little more conversational) than we usually do in our docs. I would probably be a little more direct "In order to accomodate users with prefers-reduced-motion ...."

Curious what @sarah11918 thinks here.